src/app/find-properties/[[...opa_id]]/page.tsx

```diff
@@ -29,11 +36,86 @@ const MapPage: FC = () => {
     useState<MapGeoJSONFeature | null>(null);
   const [isStreetViewModalOpen, setIsStreetViewModalOpen] =
     useState<boolean>(false);
-  const [streetViewLocation, setStreetViewLocation] =
-    useState<Position | null>(null);
+  const [streetViewLocation, setStreetViewLocation] = useState<Position | null>(
+    null
+  );
   const [smallScreenMode, setSmallScreenMode] = useState("map");
   const prevRef = useRef("map");
   const sizeRef = useRef(0);
+  const linkedPropertyRef = useRef<string | null>(null);
+  const [initialViewState, setInitialViewState] = useState<ViewState>({
+    longitude: -75.15975924194129,
+    latitude: 39.9910071520824,
+    zoom: 13,
+    bearing: 0,
+    pitch: 0,
+    padding: { top: 0, bottom: 0, left: 0, right: 0 },
+  });
+  const [linkedPropertyParsed, setLinkedPropertyParsed] = useState(false);
```

Comment:
this is 1000% a nit lol but i prefer to have boolean states start with `is`, so would be nice to have this as `isLinkedPropertyParsed`

Reply:
fixed :)